recipe/meta.yaml

```
@@ -1,58 +1,47 @@
 {% set name = "scikit-bio" %}
-{% set version = "0.5.6" %}
-{% set sha256 = "48b73ec53ce0ff2c2e3e05f3cfcf93527c1525a8d3e9dd4ae317b4219c37f0ea" %}
-{% set build_num = "0" %}
+{% set version = "0.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - scikit-bio-pr-1719-backport.patch
-    - scikit-bio-pr-1720-backport.patch
-    # See: https://github.com/biocore/scikit-bio/issues/1757
-    - skip-broken-tests.patch   # [py > 36]
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/scikit_bio-{{ version }}.tar.gz
+  sha256: 771093ebfd154f02b9d88dcf81997737ce804e384bb3ab1cd389dfc60d566d11
 
 build:
-  # non-x86 (currently) not supported due to SSE intrinsics use
-  skip: true  # [win or py2k or not x86]
-  number: {{ build_num }}
-  script: python -m pip install --no-deps --ignore-installed .
+  # Latest array-api-compat is supported only from python 3.10
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - patch     # [not win]
-    - m2-patch  # [win]
+    - {{ stdlib('c') }}
   host:
     - python
     - pip
     - cython
-    - numpy-devel {{ numpy }}  # [not (osx and arm64) and py<=39]
-    - numpy  {{ numpy }}       # [(osx and arm64) or py>39]
+    - numpy {{ numpy }}
   run:
     - python
-    - lockfile >=0.10.2
-    - cachecontrol >=0.11.5
     - decorator >=3.4.2
-    - ipython >=3.2.0
-    - matplotlib-base >=1.4.3
     - natsort >=4.0.3
-    - pandas >=1.0.0
-    - scipy >=1.3.0
-    - scikit-learn >=0.19.1
-    - hdmedians >=0.13
-    - pytest
+    - pandas >=1.5.0
+    - scipy >=1.9.0
+    - requests >=2.20.0
+    - h5py >=3.6.0
+    - biom-format >=2.1.16
+    - statsmodels >=0.14.0
+    - patsy >=0.5.0
+    - array-api-compat >=1.3
+    - llvm-openmp  # [osx]
```

Something that isn't obvious, when I tried to build on a Docker instance I was getting some overlinking errors -- which are just warnings on PBP (not ideal) -- for which I needed:

```suggestion
    - llvm-openmp  # [osx]
    - _openmp_mutex  # [linux]
```